docs/guides/runtime-options-overview.mdx

```diff
@@ -944,11 +944,12 @@ Due to differences in the device compilation process, certain runtime features c
 
   <TabItem value="Fractional" label="Fractional gates">
   Incompatible with:
-  - Dynamic circuits
   - Gate twirling
   - PEA
   - PEC
 
+  Compatible with dynamic circuits when using `qiskit-ibm-runtime` v0.42.0 or later.
```

Comment:
Should I specify why, since Jessie gave the reason that it's necessary to do this to get the correct backend target? Or does this suffice?